project.clj

```diff
@@ -12,7 +12,9 @@
   :exclusions [org.clojure/clojure]
 
   :dependencies [[org.clojure/clojure "1.6.0"]
+                 [org.clojure/java.jdbc "0.3.7"]
                  [cheshire "5.3.1"]
+                 [org.postgresql/postgresql "9.3-1100-jdbc41"]
```

Comment:
Although I suppose we could expect any consuming service to rely on the JDBC drivers that they need?